spec/factories/in_person_enrollments.rb

```diff
@@ -39,9 +39,8 @@
     end
 
     trait :with_notification_phone_configuration do
-      after(:build) do |enrollment|
-        enrollment.notification_phone_configuration =
-          FactoryBot.build(:notification_phone_configuration)
+      after(:create) do |enrollment|
+        create :notification_phone_configuration, in_person_enrollment: enrollment
       end
```

Comment:
could we use an assocation here so it respects the build/create level of the calling code?

```suggestion
      association : notification_phone_configuration
```

Reply:
done in 2d138130ca779528c2ee3d9ac6f27a1ecd35aa1b